stories/components/AdminLessonCard.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import AdminLessonCard from '../../components/admin/lessons/AdminLessonCard'
+
+export default {
+  title: 'Components/AdminLessonCard',
+  component: AdminLessonCard
+}
+
+const props = {
+  lesson: {
+    title: 'Foundations of Javascript',
+    description: 'A super simple introduction to help you get started!',
+    docUrl: undefined,
```

Comment:
i dont think this field needs to be passed if it is passed as undefined. Since the type definition says it is an optional field